core/modules/entity/entity.module

```diff
@@ -256,6 +256,39 @@ function entity_view(EntityInterface $entity, $view_mode = 'full', $langcode = N
   return FALSE;
 }
 
+/**
+ * Generate an array for rendering the given entity.
+ *
+ * @param $entities
+ *   An array of entity objects to render.
+ * @param string $view_mode
+ *   A view mode as used by this entity type, e.g. 'full', 'teaser'...
+ * @param string $langcode
+ *   (optional) A language code to use for rendering. Defaults to the global
+ *   content language of the current request.
+ * @param bool|NULL $page
+ *   (optional) If set will control if the entity is rendered: if TRUE
+ *   the entity will be rendered without its title, so that it can be embeded
+ *   in another context. If FALSE the entity will be displayed with its title
+ *   in a mode suitable for lists.
+ *   If unset, the page mode will be enabled if the current path is the URI
+ *   of the entity, as returned by entity_uri().
+ *
+ * @return
+ *   The renderable array, keyed by the entity type and by entity ID. If
+ *   there is no information on how to view an entity, FALSE is returned.
+ */
+function entity_view_multiple($entities, $view_mode = 'full', $langcode = NULL, $page = NULL) {
+  $entity_type = $entity->entityType();
```

Comment:
Note `$entity` is undefined. 
Ideally should be `entity_view($entity_type, $entities, $view_mode = 'full', $langcode = NULL, $page = NULL)` or we'd need to reset to get the first entity, and/or `foreach` to be sure its not a mix of entities, which is messier I think.

Reply:
oops